### DISCUSSION TITLE
I am making super short Prodigy hacking tutorials on YouTube

### DISCUSSION THREAD

**Discussion by ghost** [Prodigy-Hacking/ProdigyMathGameHacking#590] [(A) Anything]
(copied from [my issue](https://github.com/Prodigy-Hacking/ProdigyMathGameHacking/issues/587) that went super off-topic because of this kid trying to ask how to make a proxy- this would fit better in Discussions anyway, not Issues)

Here's the link: https://www.youtube.com/watch?v=AZzgQRV6O4k

I plan to make more videos about Prodigy, too. I'll probably make some more hacking tutorials as well.

I feel like the short video length helps because all the top results on YT are currently super long to increase ad revenue, but that isn't the best for people trying to learn.

**Comment by PatheticMustan** [comment]
Awesome! As long as you use the proper credit, I think this would be really cool

**Reply by ghost** [reply]
yes, i put proper credit in the description and it says to go to this repository in the video

**Reply by PatheticMustan** [reply]
![image](https://user-images.githubusercontent.com/35382971/109830012-13e29400-7c0c-11eb-9f5f-dc870e761f03.png)
<3 <3 <3
